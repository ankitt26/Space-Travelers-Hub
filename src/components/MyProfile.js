const MyProfile = () => (
  <main className="Myprofile">
    <div>
      <h2> My Mission</h2>
      <div className="reserved_missions">
        <h3>Mission 1</h3>
        <h3>Mission 1</h3>
        <h3>Mission 1</h3>
      </div>
    </div>
    {/* Rocket section */}
    <div>
      <h2>My Rockets</h2>
      <ProfileRocket />
    </div>
  </main>
);

const ProfileRocket = () => (
  <div className="reserved_rockets">
    <h3>Falco 9</h3>
    <h3>Falco 9</h3>
    <h3>Falco 9</h3>
  </div>
);

export default MyProfile;
