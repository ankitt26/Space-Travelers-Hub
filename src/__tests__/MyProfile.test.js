import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import Store from "../Redux/store";
// import { MemoryRouter } from "react-router";
import MyProfile from "../components/MyProfile";

it("MyProfile component test", () => {
  const tree = renderer
    .create(
      <Provider store={Store}>
        <MyProfile />
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
