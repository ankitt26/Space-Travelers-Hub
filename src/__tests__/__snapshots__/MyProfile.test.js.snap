// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyProfile component test 1`] = `
<main
  className="MyProfile"
>
  <div
    className="mission_section"
  >
    <h2
      className="heading2"
    >
      My Missions
    </h2>
    <div
      className="reserved"
    />
  </div>
  <div
    className="rocket_section"
  >
    <h2
      className="heading2"
    >
      My Rockets
    </h2>
    <div
      className="reserved"
    />
  </div>
</main>
`;
